README.md

```diff
@@ -1,38 +1,27 @@
-gavant-ember-table
-==============================================================================
+# gavant-ember-table
 
-[Short description of the addon.]
+Data tables built using [ember-table](https://github.com/Addepar/ember-table)
 
+## Compatibility
 
-Compatibility
-------------------------------------------------------------------------------
+- Ember.js v3.8 or above
+- Ember CLI v2.13 or above
+- Node.js v8 or above
 
-* Ember.js v3.8 or above
-* Ember CLI v2.13 or above
-* Node.js v8 or above
-
-
-Installation
-------------------------------------------------------------------------------
+## Installation
 
 ```
 ember install gavant-ember-table
 ```
 
-
-Usage
-------------------------------------------------------------------------------
+## Usage
 
 [Longer description of how to use the addon in apps.]
```

Comment:
not super high priority, but I would add at least a basic example of how to use the table, and then probably just provide a link to the official ember-table docs website, since 99% of that applicable here. You can check out some of our other more recent addons for examples on how we wrote usage guides.